OpenRA.Mods.Common/Traits/BotModules/SquadManagerBotModule.cs

```diff
@@ -374,6 +377,9 @@ void AssignRolesToIdleUnits(IBot bot)
 				minAttackForceDelayTicks = Info.MinimumAttackForceDelay;
 				CreateAttackForce(bot);
 			}
+
+			if (respondToAttackCooldown == MaxRespondToAttackCooldown)
```

Comment:
```suggestion
			if (respondToAttackCooldown-- == MaxRespondToAttackCooldown)
```

maybe to keep the timer localised

Reply:
neat

Reply:
Fixed